docs/tr1/CHANGELOG.md

```diff
@@ -25,12 +30,14 @@
 - fixed the `/pos` command not showing demo and cutscene titles
 - fixed the embedded bats fix causing problems inside rooms with trapdoors (regression from 4.6)
 - fixed cutscene music looping (#2591, regression from 4.8)
+- fixed the distance travelled stat displaying the wrong value when over 1000m (#2659)
```

Comment:
This isn't needed for TR1 as it's unreleased, but the line is needed in TR2's readme.